src/frontend/posts_page/components/SetTextEditor.tsx

```diff
@@ -0,0 +1,48 @@
+import React from 'react';
+
+interface MyProps {
+}
+
+interface MyState {
+  rich: boolean,
+  plain: boolean,
+}
+
+class SetTextEditor extends React.Component<MyProps, MyState> {
```

Comment:
Same suggestion as the `SetPublic.tsx` comment

Reply:
This has been changed this morning. See it is outdated

Reply:
Resolved